docs/libraries/mst-form-type.md

```diff
@@ -0,0 +1,95 @@
+# MST Form Type
+
+npm: https://www.npmjs.com/package/mst-form-type
+
+## Introduction
+
+Librarys like Ant Design has a build-in Form component that can hold fields status and validation rules. This component houses its own store to sustain internally related field statuses and can be conveniently used independently. integrating Form component with MobX State Tree models can pose significant challenges as business logic grows intricately complex.
```

Comment:
```suggestion
Libraries like [Ant Design](https://ant.design/) have a built-in `Form` component that holds field status and validation rules.  Integrating a `Form` component with MobX State Tree models can pose significant challenges as business logic become more complex.
```

**suggestion:** I'd like to fix some spelling and grammar errors here, and also clarify some of the sentences.